catkit2/services/thorlabs_cube_motor_kinesis/thorlabs_cube_motor_kinesis.py

```diff
@@ -88,7 +88,11 @@ def open(self):
         """
         # Build the device list and verify that the device is connected.

```

Comment:
Since we're on it, could you add a word to this already existing comment?
```suggestion
        # Build the device list and verify that the requested device is connected.
```